### DISCUSSION TITLE
Story does not render webcomponents

### DISCUSSION THREAD

**Discussion by razvancurcudel** [storybookjs/storybook#15818] [Help]
Hello, 

I use Storybook for rendering my custom webcomponents. I use it with NX. 
With version 5 of storybook, all seemed to worked fine. After an upgrade of NX and thus an update to Storybook to version 6, I got some problems rendering my webcomponents. I receive the below error.

```
Expecting an HTML snippet or DOM node from the story: "Basic" of "ComponentX".
Did you forget to return the HTML snippet from the story?
Use "() => <your snippet or node>" or when defining the story.
```

Also, some of the stories, despite the fact that they export multiple functions with different states of the webcomponent, in the storybook appears just one.

Storybook version: "@storybook/web-components": "^6.3.6"

**Comment by jonniebigodes** [comment]
@razvancurcudel can you create a small reproduction following these [guidelines](https://storybook.js.org/docs/react/contribute/how-to-reproduce) and factoring in NX into them so that the issue could be triaged a bit better and possibly someone can help you solve it?

Thanks in advance.

Stay safe
